src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const initialKeys = Object.keys(robot);
+
+  for (let i = 0; i < initialKeys.length; i++) {
```

Comment:
Nested for loops for checking duplicate values are not optimal. Consider using a Set for checking duplicate values in an array as it is more efficient.